text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,505 @@
+---
+rfc pr: [#347](https://github.com/aws/aws-cdk-rfcs/pull/347)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/66
+---
+
+# CDK Self Managed Stack Set Support
+
+As a CDK user, I would like to define stack set infrastructure through native CDK constructs and create/update self-managed stack sets through CDK deploy command.
+
+* **Original Author(s):** @linsona
+* **Tracking Issue**: #66
+* **API Bar Raiser**: @skinny85
+
+## Working Backwards
+
+### CHANGELOG
+
+- core: New Stack Set core type 
+- assembly: New stack set artifcat type and schema
+- cli: `cdk deploy` collects and creates/updates stack set 
```

Comment:
These should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).

Reply:
Reformatted